docs/05-command-line.md

```diff
@@ -218,13 +218,13 @@ When running a file with and without line numbers, line numbers take precedence.
 
 ## Resetting AVA's cache
 
-AVA may cache certain files, especially when you use our [`@ava/babel`](https://github.com/avajs/babel) provider. If it seems like your latest changes aren't being picked up by AVA you can reset the cache by running:
+AVA itself does not cache files unless used with our [`@ava/babel`](https://github.com/avajs/babel) provider. However, if it seems like your latest changes aren't being picked up by AVA you can try resetting the cache by running:
 
 ```console
 npx ava reset-cache
 ```
 
-This deletes all files in the `node_modules/.cache/ava` directory.
+This notifies you if cache files within the `node_modules/.cache/ava` directory are deleted or if there are no cache files to be deleted.
```

Comment:
I think the original copy was fine. If it logs that no files were deleted that's OK but that there's no need to mention that here.

Reply:
Agreed, I was trying to be more explicit at the time but I realize now that this extra info is probably unnecessary.